### PR TITLE
Update Distributors.geojson

### PR DESCRIPTION
**Pull request by GatorYK**
Active Robots from UK added

### CODE REVIEW

**Comment by GatorYK** [comment]
Active8 Robots added

File: Distributors.geojson [hunk 1]
```diff
@@ -143,6 +143,22 @@
...
+        "Organization": "Active8 Robots",
+        "Role": "Distributor",
+        "Website": "www.active8robots.com",
+        "Product": "Kobuki & Turtlebot 2"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          -2.506692, 
++          51.253259
```
Comment (bit-pirate) [inline Distributors.geojson]: Looks like a "+" sneaked in here. Please remove it.

**Comment by GatorYK** [comment]
'+' has been deleted.

**Comment by bit-pirate** [comment]
All good now.

**Pull request merged**